scripts/release/conf/conf/app.conf

```diff
@@ -0,0 +1,99 @@
+component_name = service_center
```

Comment:
we do not need to make a copy in source code, can 'cp' one copy from etc/conf/app.conf and 'sed' command to change the app.conf when execute release script.